ballot/admin/referendum.py

```diff
@@ -2,4 +2,10 @@
 
 from ..models import referendum as models
 
-admin.site.register(models.Referendum)
+
+class ReferendumAdmin(admin.ModelAdmin):
+    fields = ('name', 'number', 'ballot',
+              'website_url', 'facebook_url', 'twitter_url')
```

Comment:
I didn't see any other way as well. I think as long as we add the `assert` I wrote to each admin where we do this reordering, or add a test for each admin where we do this reordering, it's fine--we'll at least notice if a field is added/removed and the admin not updated.